README.md

```diff
@@ -0,0 +1,6 @@
+**Todos**
+
+1. Implement a interpolater to read the file
```

Comment:
First thought you had to define what kind of input for the file that you're going to take how are you going to ensure that we're going to get the file that we need for the input.